examples/get_iss_data.py

```diff
@@ -10,6 +10,7 @@
 from skimage.io import imread, imsave
 from slicedimage import ImageFormat, ImagePartition, Tile, Writer
 
+from starfish.image import Keys
```

Comment:
@ambrosejcarr what do you think of the name Keys for this? at the risk of starting this over, i prefer Dims or Axes 😁